spec/models/analytics_events_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Analytics do
```

Comment:
I'm not sure I understand the inclusion of this spec. Is this meant to offer a pattern for tracking analytics events? That may make more sense in a location that matches where in the code these live. In `spec/services`. And I think we do have good coverage of the behavior of the analytics tooling in `spec/services/analytics_spec.rb`.

Reply:
This spec was added when we were working on FSMv2 to ensure that as we re-wrote code, we didn't lose the logging of events. This was particularly important since for many of the events, the only testing that they were logged came in the controller specs, and since we were getting rid of the controllers, those specs were being deleted.

Reply:
@solipet: I know that is true for `spec/features/idv/analytics_spec.rb`, but this looks like an entirely new test file

Reply:
🤦  _nevermind..._